dashboard/test/ui/features/level_group_multi_page_dots.feature

```diff
@@ -11,6 +11,7 @@ Background:
   And I wait to see ".nextPageButton"
   And element ".nextPageButton" is visible
 
+@skip
```

Comment:
Intentional to leave this `@skip` tag in?

Reply:
definitely not, thanks for catching!